.github/workflows/compile_engine.yml

```diff
@@ -0,0 +1,53 @@
+name: Release binaries for circus-engine
+
+on:
+  push:
+    tags:
+      - '*'
+
+jobs:
+  release:
+    if: startsWith(github.ref, 'refs/tags/')
+    strategy:
+      fail-fast: false
+      matrix:
+        os:
+          - macos-latest
+          - ubuntu-latest
+          # - windows-latest
+        ocaml-compiler: [4.14.x]
+        
+    runs-on: ${{ matrix.os }}
+    permissions:
+      contents: write
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - uses: ocaml/setup-ocaml@v2
+        with:
+          ocaml-compiler: ${{ matrix.ocaml-compiler }}
+
+      - uses: actions-rs/toolchain@v1
+        id: rust_toolchain
+        with:
+          toolchain: stable
+          
+      - run: cargo install --path cli
+
+      - run: opam install . --deps-only
```

Comment:
How do we compile for m1 here? Is there some `--target` in opam/dune?

Reply:
I have absolutely no idea :/
Also, the binaries I produce are not static, and it seems not that easy to make OCaml output fully static binaries...

That's why for now I just compile to JS, I think that's a good fallback solution (`nodejs` is available basically on every plateform)

Reply:
ok, let's use this for now and discuss how to do this.